patches/0001_js_mse_eme_mvt.patch

```diff
@@ -402,28 +404,35 @@ index c040757..bf5192a 100644
      document.body.appendChild(createElement('div', 'testview'));
  
      var div = document.getElementById(this.divId);
-@@ -150,12 +151,17 @@ function TestView(testSuiteVer) {
+@@ -149,14 +150,21 @@ function TestView(testSuiteVer) {
+       switchDiv.lastChild.exec = switchDiv.lastChild.onclick;
      }
      for (var i = 0; i < selectors.length; ++i) {
-       switchDiv.appendChild(document.createTextNode(selectors[i].text));
+-      switchDiv.appendChild(document.createTextNode(selectors[i].text));
 -      var select = document.createElement('select');
-+      var select = document.createElement('span');
-+      select.id = selectors[i].id;
-       for (var j = 0; j < selectors[i].optionTexts.length; ++j) {
+-      for (var j = 0; j < selectors[i].optionTexts.length; ++j) {
 -        select.appendChild(createOption(selectors[i].optionTexts[j],
```

Comment:
createOption is now unused, please remove our related changes from .patch file. Do not remove whole function, just what was added by us (in order to minimize the patch file):
```
-var createOption = function(text, value) {
+var createOption = function(text, value, isSelected = false) {
   var option = document.createElement('option');
   option.text = text;
   option.value = value;
+  isSelected && (option.selected = 'selected');
   return option;
 };
```